tests/test_permutation.py

```diff
@@ -0,0 +1,241 @@
+import pytest
+from keum.permutation import Poseidon
+from keum.bn254 import Fr
+
+
+# Parameters and expected results generated via: https://extgit.iaik.tugraz.at/krypto/hadeshash/-/tree/master
+def test_poseidon_bn254_params():
```

Comment:
I like the interface to instantiate a new Poseidon permutation! It is clean! Nice!